data-prepper-core/src/integrationTest/java/com/amazon/dataprepper/integration/EndToEndRawSpanTest.java

```diff
@@ -135,6 +137,66 @@ public void testPipelineEndToEnd() throws InterruptedException {
         );
     }
 
+    @Test
+    public void testPipelineCompatibilityEndToEnd() throws InterruptedException {
```

Comment:
Seeing as this is very similar to the existing `testPipelineEndToEnd()` test - was it copied with the intent of making changes to the 2 methods individually? Or can they be kept as a single test for now?

Reply:
> was it copied with the intent of making changes to the 2 methods individually?
Somewhat. It is similar except the diff you noticed below.